Makefile

```diff
@@ -8,6 +8,11 @@ test:
 cov:
 	pytest --cov=kweb
 
+run:
+	export KWEB_FILESLOCATION=/tmp/gdsfactory
+	cd src/kweb
+	uvicorn --reload default:app
+
```

Comment:
why readd the run? The readme clearly states how to run it. Not everything needs to be a makefile. Plus I don't want to tie kweb to gdsfactory. If we do defaults like this, suddenly we won't notice in gdatasea if we don't properly unset/change a default.

Reply:
Also, this won't work under windows (admittedly the README has that issue as well)

Reply:
I can also remove this